src/blueprints/tv2/action-factories/tv2-transition-effect-action-factory.ts

```diff
@@ -201,7 +204,7 @@ export class Tv2TransitionEffectActionFactory {
   private updateTimelineObjectsWithTransitionEffect(action: Tv2TransitionEffectAction, piece: Piece): Tv2TransitionEffectAction {
     const sourceInput: number | undefined = this.videoMixerTimelineObjectFactory.findProgramSourceInputFromPiece(piece)
     if (!sourceInput) {
-      console.log('Can\'t find a Program SourceInput to put the Transition Effect on')
+      this.loggerService.warn('Can\'t find a Program SourceInput to put the Transition Effect on')
```

Comment:
In this task, it would be nice if the messages themself are checked as well along with adding potential data. In this case, program, source input and transition effect should not be capitalized. And information about the action and piece would be nice to have included. So this line would be something like:
```typescript
      this.logger.data({ action, piece }).warn('Can\'t find a program source input to put the transition effect on.')
```

Reply:
Will be going through monday making the changes for this, and potentially added some more log statements here and there. 

For example are we currently logging (info) in the `CachedRundownRepository.getRundown` when we retrieve the rundown from the database, but do not have any similar log in the other Cache versions (Segment, Part, Pieces). Think i am going to add similar log statement there, as debug level. Propably going to go through an check if we have any other similar areas where one of a set is logging something, that the others doesn't. Not aiming to add tons of logs, just so similar ones logs at the same area.

Reply:
As @LindvedKrvang mentions, we don't need a lot of debug logs, since they are for debugging purposes. We want to keep our logging tight – errors, warnings and info logs are what is used in production and these should have the necessary information to help out debugging an incident.

Reply:
Have not added any extra log statements as requested.

Should i remove the one in `CachedRundownRepository`?
It is a debug level one, which should only ever be written ones per rundown, so i don't see the harm in keeping it.
It has already revealed the frontend bug regarding cleanup, so we know it is well placed and usefull.